### ISSUE TITLE
Mp3 Music setPosition works wrong

### ISSUE THREAD

**Issue by denisk20** [libgdx/libgdx#6628]
#### Issue details
I have an mp3 track of 135 seconds. When I set the music position to 134 seconds I expect the music to play for 1 second. Instead it plays for almost 10 seconds. 

#### Reproduction steps/code

		music = Gdx.audio.newMusic(Gdx.files.internal("24STRINGSBASSSOLO1.mp3")); // a 135 sec. track
		music.play();

		Timer.instance().scheduleTask(new Timer.Task() {
			@Override
			public void run() {
				music.setPosition(134); // expect it to play for only 1 second, instead it plays almost 10.
			}
		}, 0.1f);

Sample repository is here https://github.com/denisk20/test-music-position

#### Version of libGDX and/or relevant dependencies
1.10.0

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [x ] MacOS

**Comment by denisk20** [comment]
There's no such issue for ogg files.

**Comment by Frosty-J** [comment]
Tested on Linux - same issue. `getPosition()` appears accurate if you let the track play all the way through without seeking, but `setPosition()` seems to cause it to become more out-of-sync the further you seek.

I tried various re-encodes with different bitrates (including variable), sample rates and joint/stereo/mono, both with Audacity and ffmpeg's libmp3lame, none of which made any difference.

No issues when running an HTML build in the browser.

While it's for Android, #5693 sounds related:
> According to [this](https://github.com/google/ExoPlayer/issues/328#issuecomment-168532148), this is a known issue for MP3 files (linked project isn't technically the MediaPlayer bundled with Android, but I'll assume the seeking behavior is similarly implemented). Converting the file to WAV or OGG fixes the issue, which is good enough for me.

Obviously this must be possible to fix, as media players such as Audacious and VLC have no such issue. But I don't have any ideas for how - all I can really come up with is the idea to make it spew warnings if the current position is greater than the track's duration.
